api/test/integration/env/docker-compose.yml

```diff
@@ -226,6 +205,11 @@ services:
     build:
       context: ./base/nginx-lb
     image: integration_test_nginx-lb
+    restart: always
```

Comment:
I found an edge case in which the nginx service exited because the workers weren't up yet, I added the restart policy and dependencies to avoid this.